docs/05-technical-reference/00-configuration-parameters/obsv-01-configpara-monitoring.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Monitoring charts
+---
+
+You find the configurable parameters for Monitoring in the [Monitoring values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/values.yaml) file and in the following subcharts:
+
+- [Grafana](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/grafana/values.yaml)
+- [Kube-state-metrics](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/kube-state-metrics/values.yaml)
+- [Prometheus Istio](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-istio/values.yaml)
+- [Prometheus Node Exporter](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-node-exporter/values.yaml)
+- [Prometheus Pushgateway](https://github.com/kyma-project/kyma/blob/main/resources/monitoring/charts/prometheus-pushgateway/values.yaml)
+
+The `values.yaml` files are fully documented and provide details on the configurable parameters and their customization.
+
+To override the configuration, modify the default values of the `values.yaml` file and [deploy them](../../04-operation-guides/operations/03-change-kyma-config-values.md).
```

Comment:
I would not mention that for every component, keep that explanation central

Reply:
Each file is meant to be able to stand alone, that's why I decided to repeat those sections.
Ideally, if the other areas agree with this approach, we can just put links to all values.yaml files into the main README (deleting all single files) and mention such generic info just once in the README.

Reply:
The sentence sounds like I should modify the values inside that file. But I should not touch the chart but provide my custom values

Reply:
good point, thanks!